src/DataTable/tests/TableHeaderCell.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import TableHeaderCell, { SortIndicator } from '../TableHeaderCell';
+
+const sortByToggleProps = { foo: 'bar' };
+const columnProps = {
+  getHeaderProps: () => ({ className: 'red' }),
+  render: () => 'Title',
+  isSorted: false,
+  isSortedDesc: false,
+  getSortByToggleProps: () => (sortByToggleProps),
+  canSort: false,
+};
+const props = {
+  column: columnProps,
+};
+
+// eslint-disable-next-line react/prop-types
+const FakeTable = ({ children }) => (<table><thead><tr>{children}</tr></thead></table>);
+
+describe('<TableHeaderCell />', () => {
+  describe('unsorted', () => {
+    const wrapper = mount(<FakeTable><TableHeaderCell {...props} /></FakeTable>);
+    it('renders a table header cell', () => {
+      const cell = wrapper.find('th');
+      expect(cell.length).toEqual(1);
+    });
```

Comment:
In a few of these tests you're nesting Cells or Rows in the context of table markup. Is that necessary? Would it cause any issues to simply test the output of this component itself since it's not rendering in browser or anything

Reply:
If I do not wrap them I get a bunch of errors when I run the tests. They don't actually fail the tests, but they do create a lot of clutter that makes it hard to see actual errors.